ReactAndroid/src/main/java/com/facebook/react/modules/appearance/AppearanceModule.java

```diff
@@ -66,6 +67,17 @@ private String colorSchemeForCurrentConfiguration(Context context) {
     return "light";
   }
 
+  @Override
+  public void setColorScheme(String style) {
+    if ("dark".equals(style)) {
```

Comment:
Can we avoid the yoda conditionals?

Reply:
Funny that you mention it, I normally don’t do this, I was trying to match existing code in RN https://github.com/facebook/react-native/blob/8d5f8804d88d7daf45766e525f8d363d937bdef5/ReactAndroid/src/main/java/com/facebook/react/modules/statusbar/StatusBarModule.java#L190